DoctrWho.Db/Interface/EnemyToEpisode.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DoctorWho.Db.Interface
+{
+    public interface IEnemyToEpisode
```

Comment:
for all interfaces, naming it more meaningful, this name looks like you have interfaces for models since its name same as model's name, please make it IEnemyToEpisodeRepository